obj/tasks.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+    nameField: '[name="taskDetailsForm.name"]',
+    nameFieldLimit: 50,
+    descriptionField: '[name="taskDetailsForm.description"]',
+    descriptionFieldLimit: 50,
+    hourlyRateField: '[name="taskDetailsForm.rate"]',
+    hourlyRateFieldLimit: 9999,
+    billToClientSelect: '.Select-value',
+    billToClientSelectedItem: '.Select-value-label',
+    createTaskButton: '.grid__column :nth-child(1) > .btn',
+    searchNameField: '.ag-header-cell:nth-child(1) .field--filter > .field__text',
+    searchDescriptionField: '.ag-header-cell:nth-child(2) .field--filter > .field__text',
+    foundNameField: '.ag-row-level-0:nth-child(1) .ag-cell-value:nth-child(1) > .ellipsis',
```

Comment:
such locators are very fragile, take a look to Martynas' code https://github.com/nikolaj-tolkaciov/sourcery-nightwatch/pull/51/files

Reply:
Improved (using xpath selectors)